controller/pkg/remoteenforcer/remoteenforcer_linux.go

```diff
@@ -172,7 +173,7 @@ func (s *RemoteEnforcer) InitEnforcer(req rpcwrapper.Request, resp *rpcwrapper.R
 
 	if err := s.setupEnforcer(req); err != nil {
 		resp.Status = err.Error()
-		return nil
+		return fmt.Errorf(resp.Status)
```

Comment:
Why not returning it as an error ? Previously the status would be completely ignored

Reply:
Yes, return an error here.

Reply:
stop evaluating resp.Status on the client side if you return an error here